webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -354,7 +354,7 @@ class ClassicTechniqueWriter(basePath : String, parameterTypeService: ParameterT
 
     val content =
       s"""# @name ${technique.name}
-         |# @description ${technique.description}
+         |# @description ${technique.description.split("\n").mkString("\n# ")}
```

Comment:
why not `replaceAll` ?

Reply:
and perhaps `[\n\r]{1,2}` ? Not sure what browser on windows do?

Reply:
according to https://stackoverflow.com/questions/454908/split-java-string-by-new-line , which point to https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html, \R should do the trick